apps/ima/behaviours/continue-report.js

```diff
@@ -64,12 +80,22 @@ module.exports = superclass => class extends superclass {
   }
 
   saveValues(req, res, next) {
-    super.saveValues(req, res, err => {
+    super.saveValues(req, res, async err => {
       if (err) {
         next(err);
       }
       req.sessionModel.set('redirect-to-summary', false);
       if (req.body['save-and-exit']) {
+        const host = req.get('host');
+        const userEmail = req.form.values['user-email'] || req.sessionModel.get('user-email');
+        const token = tokenGenerator.save(req, userEmail);
+        try {
+          await notifyClient.sendEmail(templateId, userEmail, {
+            personalisation: getPersonalisation(host, token, req, req)
```

Comment:
It doesn't look to me like `getPersonalisation` needs the `req` object passing into it.

Reply:
twice :)

Reply:
thanks, have removed this now